src/components/Person.jsx

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+import './person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        { name ? `My name is ${name}` : 'Unknown'}
+      </h2>
+      <p className="Person__age">
+        { age && `I am ${age}`}
+      </p>
```

Comment:
You shouldn't render this tag if age is null
{age && (
  <p> ... </p>
)}

Reply:
Maybe I didn't  understand what you mean, but this code works. It doesn't render this tag if age is null.

Reply:
Thanks for explanation, I've understood. Fixed.